### ISSUE TITLE
Black screen after suspend

### ISSUE THREAD

**Issue by lalit12398** [elementary/greeter#400]
I've been using elementary os from the past few days and cannot wake screen after suspend.
For a temporary fix, I turned of suspend option from everywhere but still if I leave my system for more than 30mins in the lock state, somehow it goes to suspend and again black screen if I try to wake it up.

**Comment by wedsonlopes** [comment]
I had the same issue, it happens all the time the OS suspends itself by inactivity..

**Comment by locness3** [comment]
Facing this problem on a 2009 macbook pro, using nvidia's drivers. Seems like it's the fault of the greeter, since, for me, it works as expected when using pm-suspend (which only suspends but doesn't lock) instead of elementary's Sleep option, and pressing Ctrl-Alt-F7 will show the "this session is locked" screen.
Does the same apply to you @lalit12398 @wedsonlopes ?

**Comment by wedsonlopes** [comment]
yes @locness3 , using a samsung laptop x23. But my problem was fixed.

**Comment by locness3** [comment]
How?

**Comment by wedsonlopes** [comment]
I just waited for updates, I didn't have to do nothing.

**Comment by lenemter** [comment]
Closing as stale
